projects/angular/src/forms/common/wrapped-control.ts

```diff
@@ -100,6 +115,22 @@ export class WrappedFormControl<W extends DynamicWrapper> implements OnInit, OnD
     }
   }
 
+  ngDoCheck() {
+    if (this.differ) {
+      const changes = this.differ.diff(this.ngControl);
+      if (changes) {
+        changes.forEachChangedItem(change => {
+          if (
+            (change.key === CHANGE_KEYS.FORM || change.key === CHANGE_KEYS.MODEL) &&
+            change.currentValue !== change.previousValue
+          ) {
+            this.triggerValidation();
+          }
+        });
```

Comment:
```suggestion
        let validationNeeded = false;

        changes.forEachChangedItem(change => {
          if (
            (change.key === CHANGE_KEYS.FORM || change.key === CHANGE_KEYS.MODEL) &&
            change.currentValue !== change.previousValue
          ) {
            validationNeeded = true;
          }
        });

       if (validationNeeded) {
         this.triggerValidation();
       }
```

Should we do something like this to prevent validation from being trigger more than once? Not sure if that's a possible issue or not.

Reply:
It would trigger only if the binded control is changed and since I'm tracking only that the items in `changes` will be always 1.